Cargo.toml

```diff
@@ -12,7 +12,7 @@ exclude     = [
 ]
 
 [dependencies]
-libc = { git = "https://github.com/rust-lang/libc" }
+libc = { version = ">=0.2.38", git = "https://github.com/rust-lang/libc", branch = "master" }
```

Comment:
This change is unnecessary and can be reverted.  Cargo will always try to use the latest version when you use a "git" specification.  If the included `libc` version in your workspace seems behind, then you should run `cargo update`.